2022-finalproject/cjj/work.md

```diff
@@ -0,0 +1,17 @@
+## 回答
+
+### 问题一
+* 使用 "git restore --staged <文件>..."
+* git reset HEAD +filename
```

Comment:
Do you want to use these two commands together to solve the problem, or can either do?